libopenage/gamestate/world.h

```diff
@@ -1,11 +1,51 @@
-// Copyright 2018-2018 the openage authors. See copying.md for legal info.
+// Copyright 2018-2023 the openage authors. See copying.md for legal info.
 
 #pragma once
 
-namespace openage::gamestate {
+#include "util/path.h"
 
+namespace openage {
+
+namespace renderer {
+class RenderFactory;
+}
+
+namespace gamestate {
+class GameEntity;
+
+/**
+ * Entity for managing "physical" things (units, buildings) inside
+ * the game.
+ */
```

Comment:
i think `Universe` still makes sense to separate all the terrain stuff (which is less complex now but could include pathing/mapinfo/etc. later on) from the game entities.

Reply:
just from a wording perspective the universe contains worlds and players, the world contains terrains and units, and we should try to match intuitive meanings and responsibilities :)
so a universe handling terrain but not the world is kinda badly named then.

Reply:
i guess my thought process is rooted in astronomy too much because I think of "universe" as something very physical :D but I think you're right and it can be confusing.

though I wouldn't change it in this PR and just add a TODO for later. we'll have some shuffling of structure around anyway, when we're addressing the gamestate next.